content/en/Getting started/glossary.md

```diff
@@ -293,6 +293,10 @@ with [white-box](#white-box-testing) and [black-box testing](#black-box-testing)
 
 Also known as "translucent-box testing."
 
+## Graylisting
```

Comment:
@molfinn Both variants work, but let's use "graylisting" because "gray" is more frequent in American English. This is really minor IMO. Reference: https://www.merriam-webster.com/words-at-play/gray-vs-grey-usage-difference